client/src/TextInput.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from "react";
+
+var inputStyle =
+  "bg-gray-600 placeholder-gray-400 px-5 py-3 rounded text-lg border border-gray-400";
+
+class TextInput extends Component {
+  render() {
+    return (
+      <div>
+        <div className="mt-5 mb-2">
+          <input
+            className={inputStyle}
+            type="text"
+            placeholder="Email address"
+          />
+        </div>
+        <div>
+          <input className={inputStyle} type="text" placeholder="Password" />
```

Comment:
By making this component contain both text inputs, we are actually making it less reusable. If we make this just for one single text input, then we could actually pass in props for the `placeholder`, and it's instantly more valuable as a component.

(Sorry if the acceptance criteria was misleading)